styleguide/docs/code_conventions.md

```diff
@@ -42,8 +42,107 @@ Class names and hierarchy follow the [BEM (Block Element Modifier)](http://getbe
+### Layouts
+
+The layouts molecules are now deprecated. To give structure to organisms and templates, please follow the grid and columns method outlined below.
+
+
+### Grids and columns
+
+#### Basics
+
+To use columns to control a template or organism's structure, start by adding an element with a `.grid` or `.container-with-grid` class (we'll get to the difference a little later). This grid instantiates a flexbox wrapper for descendant elements.
+
+Then add elements with `.col-width-x` classes, where 'x' should be replaced by the number of columns you want the element to span. We use a 12 column grid, so for any given `.grid` container, make sure that the `.col-width-x` numbers add up to a total of 12.
+
+**Example:**
+
+```
+<a href="#" class="grid">
+  <div class="col-width-8">
+    {% include '09-text.twig' %}
+  </div>
+  <div class="col-width-4">
+    {% include '09-text.twig' %}
+  </div>
+</a>
+```
+
+The `col-width-x` classes include default left and right padding. At our mobile breakpoint, they default to `width:100%` (in other words, multiple columns collapse into a single-column layout).
+
+If you need to specify different gutters, or use a different mobile behavior for your pattern , apply the `grid()` and the `grid__unit--cols(x)` _mixins_ to your named classes rather than using the `.grid` and `.col-width-x` _classes_.
+
+**Example:**
+
+Markup
+
+```
+<a href="#" class="news-section">
+  <div class="news-section_left">
+    {% include '09-text.twig' %}
+  </div>
+  <div class="news-section_right">
+    {% include '09-text.twig' %}
+  </div>
+</a>
+```
+
+SCSS
+
+```
+.news-section {
+  @include grid(); 
+}
+.news-section_left {
+  @include grid__unit--cols(8);
+}
+.news-section_right {
+  @include grid__unit--cols(4);
+}
+```
+
+
+**Avoid** mixing usage of the `.grid` _class_ and the `grid__unit--cols(x)` _mixin_. Similarly, do not combine the `grid()` _mixin_ with the `.col-width-x` _classes_. Parent and child elements should be consistent--use either classes **or** mixins but **not both**. This is to maintain clarity and make it more intuitive for a developer to see how the grid's being implemented in a given pattern.
+
+**Do not do this:**
+
+Markup
+
+```
+<a href="#" class="news-section grid">
+  <div class="news-section_right">
+    {% include '09-text.twig' %}
+  </div>
+  <div class="news-section_left">
+    {% include '09-text.twig' %}
+  </div>
+</a>
+```
+
+SCSS
+
+```
+.news-section_right {
+  @include grid__unit--cols(8);
+}
+.news-section_left {
+  @include grid__unit--cols(4);
+}
+```
+
+**Note:** Under the hood, both the `.grid` class and the `@grid` mixin leverage [sass-grid](https://github.com/digitaledgeit/sass-grid), which is a very lightweight, flexbox-based grid system. You might want to look at how this grid works, but you should never need to work with it directly.
+
+#### `.container` vs `.container-with-grid` vs `.grid`
```

Comment:
I love this section - the names are so similar and this helps to clarify. How about moving it up higher? Perhaps under Basics there is a section called "Choosing the right wrapper" or something along those lines.